features/add_dishes.feature

```diff
@@ -7,14 +7,18 @@ Feature: Add dishes to menu
    Given the following restaurants exist
     | name        |
     | NisseKebbab |
-   And the following menus exist
+    | AmberBurger |
```

Comment:
Oh, so you removed the restaurant with an empty menu. Well, I'm still curious how the system reacts to an empty mnu, what do you display? A scenario for that could be in place.